apparmor.d/profiles-a-f/briar-desktop

```diff
@@ -0,0 +1,97 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Roman Beslik <me@beroal.in.ua>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
```

Comment:
I have just updated to abi 4. Therefore, from now on:
```
abi <abi/4.0>,
```

Reply:
Compiling [the most recent version of AppArmor](https://aur.archlinux.org/packages/apparmor-git) failed. I don't know where I can get the version of AppArmor supporting ABI 4.0. 😢